src/Codeception/Lib/Connector/Yii2.php

```diff
@@ -275,6 +275,16 @@ protected function persistDb()
 
     private function mockAssetManager()
     {
-        $this->app->set('assetManager', Stub::make('yii\web\AssetManager', ['bundles' => false]));
```

Comment:
I know that was a dirty fix but it worked for some cases. At least for me it didn't create `@webroot`. Are we 100% sure that `@webroot` won't be created ever again with this patch?